recipes/pretextsnapshot/meta.yaml

```diff
@@ -0,0 +1,33 @@
+{% set name = "PretextSnapshot" %}
+{% set version = "0.0.2" %}
+{% set checksum = "bbb4ce292c13263616b24233f49859cb9d139cd84d56729ec5dd495782d87536" %}
+{% set about = "Commandline image generator for Pretext contact maps." %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://github.com/wtsi-hpag/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: {{ checksum }}
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('cxx') }} # [linux64]
+    - make
+    - clang>=9.0.0
+    - python
+
+test:
+  commands:
+    - {{ name }}
+
+about:
+  home: https://github.com/wtsi-hpag/{{ name }}
+  license: MIT
+  license_file: LICENSE
+  license_family: MIT
+  summary: {{ about }}
```

Comment:
Please put a more informative description here (possibly in a `description` section if it's long). Generally few people will know what a pretext contact map is, so at least mention HiC or something like that in there.

Reply:
I've updated the description and summary